tests/acceptance/routeable-engine-demo-test.js

```diff
@@ -277,6 +277,40 @@ test('transitionTo works properly within parent application', function(assert) {
   });
 });
 
+test('transitionToExternal transitions to the parent application from within an engine', function(assert) {
+  assert.expect(2);
+
+  visit('/routable-engine-demo/special-admin-blog-here/post/1');
+  click('.routable-post-transition-to-home-button');
+
+  andThen(() => {
+    assert.equal(currentURL(), '/');
+
+    visit('/routable-engine-demo/special-admin-blog-here/post/1');
+
+    andThen(() => {
```

Comment:
Let's avoid nesting the test helpers as it makes the tests harder to read/follow.